src/modules/error/hocs/withPayloadProvider.js

```diff
@@ -51,7 +50,7 @@ const withPayloadProvider = <S: { dispatch: Dispatch<StoreActionType> }, R> (
 
       refresh = () => {
         const props = this.props
-        if (props.status === 'routeNotInitialized' || props.status === 'loading') {
+        if (props.status === 'routeNotInitialized' || props.status === 'loading' || props.status === 'languageNotAvailable') {
```

Comment:
You could do `['routeNotInitialized', ..].includes(props.status)`. Not sure if it is better though.

Reply:
I guess, flow doesn't understand that...

Reply:
Ahh yeah okay...